test/common.py

```diff
@@ -311,7 +311,10 @@ def remove_prefix(text, prefix):
                                      munged_id)
         if subname:
             expected_file += "-" + subname
-        expected_file += ".expect"
+        if sys.platform == 'win32' and os.path.exists(expected_file + '.expect.win'):
+            expected_file += ".expect.win"
```

Comment:
Uh I don't think that's a good solution for expect tests. It will make all the diffs that touch them 2x larger + is really hard to regenerate them for devs who don't use Windows (which is pretty much everyone). This is too much of a productivity hit for us. We should just figure out a way to get consistent output on Windows